Gemfile

```
@@ -11,9 +11,13 @@ group :release do
   gem 'github_changelog_generator', '~> 1.16.4', require: false
 end
 
-# openvox on Ruby 3.3 / 3.4 has some missing dependencies
-# Will be fixed in a future openvox release
-gem 'base64', '~> 0.2' if RUBY_VERSION >= '3.4'
-gem 'puppet', '>= 7', '< 9'
-gem 'racc', '~> 1.8' if RUBY_VERSION >= '3.3'
+if ENV.fetch('IMPLEMENTATION', nil) == 'puppet'
+  # openvox on Ruby 3.3 / 3.4 has some missing dependencies
+  # Will be fixed in a future openvox release
```

Should the comment now say `puppet`?

```suggestion
  # puppet on Ruby 3.3 / 3.4 has some missing dependencies
  # Will be fixed in a future puppet release
```

Maybe remove the "will be fixed" comment?

---

good catch, fixed it